src/CoreShop2VueStorefrontBundle/Bridge/DocumentMapper/AbstractMapper.php

```diff
@@ -34,4 +35,16 @@ protected function getDateTime(int $date): string
     {
         return Carbon::createFromTimestamp($date)->format(\DateTimeInterface::RFC3339);
     }
+
+    public function find(IndexService $service, object $object): object
```

Comment:
This obsoletes `DocumentFactory`, each mapper will become its own document factory instead, allowing custom factories / finders directly inside the mapper. Not sure about the design yet though, maybe merging it into a single method might be best.